aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -2,11 +2,9 @@
 
 import numpy as np
 import openmdao.api as om
-
```

Comment:
This test should have its docstrings converted back to comments.

Reply:
done.